src/NuGet.Core/NuGet.Protocol/Utility/MetadataReferenceCache.cs

```diff
@@ -97,12 +100,28 @@ public T GetObject<T>(T input)
                 _propertyCache.Add(typeKey, properties);
             }
 
+            if (!_stringMethodsCache.ContainsKey(typeof(MetadataReferenceCache)))
+            {
+                // Doing reflection everytime is expensive so cache it for string type which is all this MetadataReferenceCache about.
+                var stringPropertyType = typeof(string);
+                var method = _metadataReferenceCacheType.GetTypeInfo()
```

Comment:
Per [coding guideline](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style) 10, we shouldn't use `var` here.

Reply:
Fixed